### ISSUE TITLE
Coinductive types?

### ISSUE THREAD

**Issue by spacekitteh** [HoTT/book#787]
Could something be said about coinductive types being derivable in the presence of inductive types in HoTT? (See http://arxiv.org/abs/1504.02949 )

**Comment by andrejbauer** [comment]
As in "add coinductive types" to the HoTT book? That would be something for 2nd edition.

**Comment by mikeshulman** [comment]
I could be convinced, but at present I don't think it would be worth including any serious development of coinductive types.  But we could add a sentence or two somewhere mentioning that they exist and giving a citation, perhaps in the notes to in chapter 5.

**Comment by awodey** [comment]
of course, Benedikt and coauthors have recently written a paper on this that could be cited.

Steve

**Comment by mikeshulman** [comment]
(which Sophie linked to in her first comment)

**Comment by awodey** [comment]
check.
